chris_backend/plugins/models.py

```diff
@@ -1,4 +1,5 @@
-
+from cgitb import lookup
+from dataclasses import field
```

Comment:
What are these two imports for?

Reply:
Must have been picked by the `VsCode` extension ... I do not think they add on what I was trying to do. I have disabled the extension

Reply:
Remove these lines. I recommend you review your `git diff` before making commits.

Reply:
Thank you, I have made chnages